criticality_score/run.py

```diff
@@ -608,6 +608,62 @@ def initialize_logging_handlers():
     logging.getLogger('').addHandler(console)
 
 
+def override_params(override_params):
+    for override_param in override_params:
+        temp = override_param.split(':',1)
+        param_name = temp[0]
+        try:
+            weight, threshold = [
+                int(i) for i in temp[1].split(':')
```

Comment:
Is it worth supporting `float` here? Particularly for `weight`.

Reply:
That's a good question! We've just started playing with this and all the defaults were integers so that's what I went with but it might be useful to make them floats indeed. We don't have enough experience to say whether we'd make use of that at this point but if you think that's worth doing I'm happy to change it to float.

Reply:
Some of the weights are `floats` already so I think it makes sense to do it for the weights at least

Reply:
You're right, I changed it to float.